lib/nats.js

```
@@ -843,6 +865,9 @@ Client.prototype.publish = function(subject, msg, opt_reply, opt_callback) {
  */
 
 Client.prototype.subscribe = function(subject, opts, callback) {
+  if (this.closed) {
+    this.emit('error', SUB_CONN_CLOSED_ERR);
```

Should we throw an exception?